src/WebJobs.Extensions.SendGrid/Config/SendGridConfiguration.cs

```diff
@@ -102,5 +107,15 @@ private static string FirstOrDefault(params string[] values)
         {
             return values.FirstOrDefault(v => !string.IsNullOrEmpty(v));
         }
+
+        // Schema for host.json 
+        private class ConfigMetadata
+        {
+            [JsonProperty("to")]
```

Comment:
Note that there's a slight name mismatch. The host.json says "to" and "from"; whereas the config says "toAddress" and "fromAddress". Having an intermediate metadata object like this clearly captures that.